pynbody/halo/hop.py

```diff
@@ -21,22 +22,28 @@ def __init__(self, sim, fname=None):
             if not os.path.exists(fname):
                 raise RuntimeError("Unable to find HOP .tag file in simulation directory")
 
-
         sim._create_array('hop_grp', dtype=np.int32)
-        sim['hop_grp']=-1
+        sim['hop_grp'] -= 1
```

Comment:
This version appears to assume that `hop_grp` is initialised to all zeros. That sounds like an unsafe assumption? Is there a reason why setting it to `-1` (the original behaviour) is a problem?

Reply:
Yes sorry this is a typo. This should go back to the original -1.